internal/clusterinfo/data.go

```diff
@@ -119,14 +119,18 @@ func (c *ClusterInfo) GetLookupdTopics(lookupdHTTPAddrs []string) ([]string, err
 
 // GetLookupdTopicChannels returns a []string containing a union of all the channels
 // from all the given lookupd for the given topic
-func (c *ClusterInfo) GetLookupdTopicChannels(topic string, lookupdHTTPAddrs []string) ([]string, error) {
+func (c *ClusterInfo) GetLookupdTopicChannels(topic string, lookupdHTTPAddrs []string) ([]string, map[string]bool, bool, error) {
```

Comment:
This is an internal API, we can change it to return a proper data structure that includes various properties (like paused state) rather than a separate map.

Reply:
that's a good suggestion

Reply:
Add new data structure: TopicMeta, TopicChannelsMeta to solve it. see code. : )